apps/sms/views/shared/test/unit/activity_handler_test.js

```diff
@@ -418,28 +418,28 @@ suite('ActivityHandler', function() {
 
       test('email in notification title when contact without name', function() {
         sinon.assert.called(sendSpy);
-        var notification = sendSpy.firstCall.thisValue;
-        assert.equal(notification.title, emailAddress);
+        var notification = sendSpy.firstCall;
+        assert.isTrue(notification.calledWith(emailAddress));
       });
     });
 
     suite('after getSelf', function() {
       var sendSpy, contactSpy;
       setup(function(done) {
-        sendSpy = this.sinon.spy(window, 'Notification');
+        sendSpy = this.sinon.spy(NotificationHelper, 'send');
         contactSpy = this.sinon.spy(Contacts, 'findByAddress');
         MockNavigatormozApps.mTriggerLastRequestSuccess().then(done, done);
       });
 
       test('a notification is sent', function() {
         sinon.assert.called(sendSpy);
-        var notification = sendSpy.firstCall.thisValue;
-        assert.equal(notification.body, message.body);
-        assert.deepEqual(notification.data, {
+        var notification = sendSpy.firstCall;
+        assert.equal(notification.args[1].body, message.body);
+        assert.deepEqual(notification.args[1].data, {
           id: message.id,
           threadId: message.threadId
         });
-        assert.equal(notification.icon, 'sms');
+        assert.equal(notification.args[1].icon, 'sms');
```

Comment:
nit: use: `sinon.assert.calledWith(sendSpy, { body: ..., data: ..., icon: ... });` (maybe you'll need calledWithMatch)